src/controllers/pages/items.js

```diff
@@ -0,0 +1,12 @@
+import { getAllModels } from '../../lib/api-client';
+import { Dispatcher } from 'consus-core/flux';
+import { hashHistory } from 'react-router';
+
+export default class ItemController {
+    static newItem() {
+        return getAllModels().then(models => {
```

Comment:
Making a new item gets all the models? shouldn't this be calling createItem in api-client?